example/extender/extender.go

```diff
@@ -49,6 +50,7 @@ func onSessionOpen(w http.ResponseWriter, r *http.Request) {
 		NamespaceInfo:  req.NamespaceInfo,
 		RevocableNodes: req.RevocableNodes,
 	}
+	klog.V(4).Infof("the snapshot of cluster %+v", *snapshot)
```

Comment:
Why add this?

Reply:
```text
example/extender/extender.go:28:5: var snapshot is unused (U1000)
```
add this code to avoid staticcheck warnings, or delete the field. i think adding a log for this is better. if you have a better method, can tell me.